programs/clearing_house/src/state/events.rs

```diff
@@ -147,6 +147,31 @@ impl Default for OrderAction {
     }
 }
 
+#[event]
+#[derive(Default)]
+pub struct LPRecord {
+    pub ts: i64,
+    pub user: Pubkey,
+    pub liquidity_type: LiquidityType,
```

Comment:
can we change     pub liquidity_type: LiquidityType,
to action: LPAction?

we use action in a lot of other places for this kind of enum

Reply:
resolved this.